src/components/componentDiff/ComponentDiffTab.js

```diff
@@ -204,6 +213,113 @@ ComponentDiffTabLoading.propTypes = {
 }
 
 
+const ExcludeSelectionDialog = ({ handleClose, diffComponents, handleExclude, handleExportDiff }) => {
```

Comment:
rather invert to include semantics and pre-select _all_ components and artefacts
rational: it is more intuitive that "selected components" will be exported